docs/products/kafka/howto/approvals.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Approvals
+limited: true
+---
+
+The approvals page allows you to manage ownership requests for Aiven for Apache Kafka resources owned by your group.
```

Comment:
Not sure about the phrase `ownership requests` but am happy to go with it if there isnt a better way to say it.
Like would `Governance requests` be better?

Reply:
Thank you for your feedback, Aindriu. I understand your concern regarding the phrase "ownership requests." We aim for clarity and ease of understanding for all users.

"Governance requests" might be more precise for those familiar with governance processes, but it could be less clear to a broader audience. To balance clarity and accuracy, how about we use "requests for Aiven for Apache Kafka resources"? This term clearly indicates the nature of the requests without assuming prior knowledge of governance terminology.
We can iterate based on feedback as well. 

What do you think?

Reply:
That sounds good to me 100% you are right!